### ISSUE TITLE
Looking for an active chapter to join. Pref. in Manila, Philippines

### ISSUE THREAD

**Issue by lloydlions** [nodeschool/organizers#533]
Hi. 

I tried searching for manila chapter here in Philippines, but it seems it is not active yet.  

Is there a way where I can join? Then do the exercises, and at the same time join with the discussion. I want to learn NodeJS, I'm studying by myself, and am not a developer. But I want to be someday. 

Thanks!

**Comment by chowdhurian** [comment]
You're correct, there doesn't seem to be a Manila chapter. The ones in the Philippines I see are [Mandaue](https://nodeschool.io/mandaue/) and [Cebu](https://nodeschool.io/cebu/) 😔

I don't know of any NodeSchool chapters that offer remote links. You do have the option to start a chapter in Manila yourself. You may start with opening an issue like #527. 

Does that sound like something you'd like to do?

**Comment by lloydlions** [comment]
Sorry for the late response, I would love to. But I'm still on the learning curve. I'm thinking of sharing this other forums such as symbianize. Maybe someone would have an interest to host.

Thanks for the reply.
